service/ProductService.js

```diff
@@ -0,0 +1,112 @@
+import axios from 'axios';
+
+const instance = axios.create({
+	baseURL: 'https://panda-market-api-crud.vercel.app',
+	headers: {
+		"Content-Type": "application/json",
+		"Accept": "application/json",
+	},
+})
+
+/**
+ * 게시물 조회
+ *
+ * @param queryParam {Object} - 쿼리 파라미터
+ * @param queryParam.page {Number} - 게시물 제목
+ * @param queryParam.pageSize {Number} - 게시물 내용
+ * @param queryParam.keyword {String} - 이미지 URL
+ */
+export const getProductList = async (queryParam = {}) => {
+	try {
+		const { page, pageSize, keyword } = queryParam;
+		const params = {
+			page: page || 1,
+			pageSize: pageSize || 10,
+			keyword: keyword || '',
+		};
+		const response = await instance.get(`/products`, { params });
+		const data = response.data;
+		const list = data.list;
+		return list;
```

Comment:
크게 const 로 나누는 것이 중요하거나 중간 const로 나눈 것을 반드시 사용할 필요가 없다면 아래와 같이 줄일 수 있을 것 같아요.
```suggestion
		return response.data.list;
```